meta-ostro/conf/local.conf.sample

```diff
@@ -35,6 +26,9 @@
 # For Intel Edison:
 #MACHINE ?= "edison"
 #
+# For BeagleBone and BeagleBone Black:
```

Comment:
Original BeagleBone was produced in 2011. Since 2013, only BBB produced. We only have BBB in testing and even only later versions of those with 4GB eMMC. (original BBB was produced with 2Gb eMMC). Our images might work without anything special on newly released BeagleBone Green (it's BBB minus hdmi and plus two groovy connectors), but we don't have any of those to verify.